bigtable/client/client_options.h

```diff
@@ -81,15 +82,56 @@ class ClientOptions {
   }
 
   /**
-   * Set the grpclb fallback timeout (in ms) for the channel.
+   * Set the grpclb fallback timeout with the timestamp [@p fallback_timeout)
+   * for the channel.
+   *
+   * This function accepts any instantiation of 'std::chrono::duration<>' for
+   * @p duration parameter convert it into milliseconds and pass it to
+   * channel_arguments. For example:
+   *
+   * @code
+   * using namespace std::chrono_literals; // C++14
```

Comment:
Why introduced the C++14 namespace if you do not use it?  Please change the code to use the chrono literals, or simply remove this line.